src/util/FitsComparatorCMD.cpp

```diff
@@ -0,0 +1,86 @@
+//
+// Created by Samuel Cappon on 3/9/20.
+//
+
+#include <util/FitsComparator.h>
+#include <core/Exceptions.h>
+#include <iostream>
+#include <boost/program_options.hpp>
+#include <boost/program_options/errors.hpp>
+
+namespace po = boost::program_options;
+
+main(int ac, char *av[]) {
+    vector<string> n;
+
+    po::options_description desc("Allowed options");
+    desc.add_options()
+    ("help", "produce help message")
+    ("input-file", po::value < vector < string >> (), "input file (2 required)")
+    ("segments,s", "compare by segments")
+    ("headers,h", "compare by headers")
+    ("hdu,u", "compare by header data units");
+
+    po::positional_options_description p;
+    p.add("input-file", 2);
+
+    po::variables_map vm;
+    po::store(po::command_line_parser(ac, av).options(desc).positional(p).run(), vm);
+    po::notify(vm);
+
+    if (vm.count("help")) {
+        cout << desc << "\n";
```

Comment:
There is no `using namespace std`, which is a good thing. Then this must be `std::cout, std::endl`